bin/src/util.rs

```diff
@@ -118,6 +119,9 @@ pub fn get_config_file_path(args: &cli::Args) -> Result<&str, UtilError> {
 
 #[cfg(target_os = "freebsd")]
 pub unsafe fn get_executable_path() -> Result<String, UtilError> {
+    use libc::{c_void, sysctl};
+    use libc::{CTL_KERN, KERN_PROC, KERN_PROC_PATHNAME, PATH_MAX};
+
     let mut capacity = PATH_MAX as usize;
     let mut path: Vec<u8> = Vec::with_capacity(capacity);
     path.extend(repeat(0).take(capacity));
```

Comment:
I would replace these line, it's simpler and avoids the `repeat` import.
```suggestion
    let mut path = vec![0; capacity];
```

Reply:
Thanks @Wonshtrum , I'll add that change now.